docs/sphinx/source/user_guide/glossary.rst

```diff
@@ -0,0 +1,195 @@
+
+    gamma_pdc
+        Module temperature coefficient. Typically in units of 1/C.
+
+    ghi
+        Global horizontal irradiance
+
+    ghi_extra
+        Horizontal irradiance at top of atmosphere (extraterrestrial)
+
+    gri
+        Ground-reflected irradiance
+
+    i_sc
+        Short circuit module current
+
+    i_x, i_xx
+        Sandia Array Performance Model IV curve parameters
+
+    latitude
+        Latitude
+
+    longitude
+        Longitude
+
+    pac, ac
+        AC power
+
+    pdc, dc
+        DC power
+
+    pdc0
+        Nameplate DC rating
+
+    photocurrent
+        Photocurrent
+
+    poa_diffuse
+        Total diffuse irradiation in plane. Sum of ground and sky diffuse.
+
+    poa_direct
+        Direct/beam irradiation in plane
+
+    poa_global
+        Global irradiation in plane. sum of diffuse and beam projection.
+
+    poa_ground_diffuse
+        In plane ground reflected irradiation
+
+    poa_sky_diffuse
+        Diffuse irradiation in plane from scattered light in the atmosphere
+        (without ground reflected irradiation)
+
+    precipitable_water
+        Total precipitable water contained in a column of unit cross section
+        from earth to top of atmosphere
+
+    pressure
+        Atmospheric pressure
+
+    relative_humidity
+        Relative humidity
+
+    resistance_series
+        Series resistance
+
+    resistance_shunt
+        Shunt resistance
+
+    saturation_current
+        Diode saturation current
+
+    solar_azimuth
+        Azimuth angle of the sun in degrees East of North
+
+    solar_zenith
+        Zenith angle of the sun in degrees
+
+    surface_azimuth
+        Azimuth angle of the surface
+
+    surface_tilt
+        Tilt angle of the surface
+
+    surface_tilt
+        Panel tilt from horizontal [°]. For example, a surface facing up = 0°,
+        surface facing horizon = 90°.
+
+    temp_air
+        Temperature of the air
+
+    temp_cell
+        Temperature of the cell
+
+    temp_dew
+        Dewpoint temperature
+
+    temp_module
+        Temperature of the module
+
+    transposition_factor
+        The gain ratio of the radiation on inclined plane to global horizontal
+        irradiation: :math:`\frac{poa\_global}{ghi}`
+
+    tz
+        Timezone
+
+    v_mp, i_mp, p_mp
+        Module voltage, current, power at the maximum power point
+
+    v_oc
+        Open circuit module voltage
+
+    wind_direction
+        Wind direction
+
+    wind_speed
+        Wind speed
+
+
+For further explanation of the variables, common symbols, and
+units, refer to the following sources from `SoDa Service <http://www.soda-pro.com/home>`_:
+
+   * `Acronyms, Terminology and Units <https://www.soda-pro.com/help/general/acronyms-terminology-and-units>`_
+   * `Plane orientations and radiation components <https://www.soda-pro.com/help/general/plane-orientations-and-radiation-components>`_
+   * `Time references <https://www.soda-pro.com/help/general/time-references>`_
```

Comment:
```suggestion
* `Acronyms, Terminology and Units <https://www.soda-pro.com/help/general/acronyms-terminology-and-units>`_
* `Plane orientations and radiation components <https://www.soda-pro.com/help/general/plane-orientations-and-radiation-components>`_
* `Time references <https://www.soda-pro.com/help/general/time-references>`_
```

It's rendering as citation, changes it to normal bulleted-list.

Reply:
I don't understand this one, the original has a shaded background and so does this? It doesn't look like a citation(?)

Reply:
You are completely right. Block quote is the right term of what it does now, sorry for the confusion.

https://bashtage.github.io/sphinx-material/rst-cheatsheet/rst-cheatsheet.html

Reply:
No problem, I appreciate you going through the docs carefully to try find these kinda things. I implemented your other suggestion to change the order of the pages btw

Reply:
Nah man, reviewing is the easy part and more entertaining than wasting time xD